packages/calypso-e2e/src/lib/components/editor-sidebar-block-inserter-component.ts

```diff
@@ -78,7 +78,10 @@ export class EditorSidebarBlockInserterComponent {
 		if ( type === 'pattern' ) {
 			locator = editorParent.locator( selectors.patternResultItem( name ) ).first();
 		} else {
-			locator = editorParent.locator( selectors.blockResultItem( name ) ).first();
+			locator = editorParent
+				.getByRole( 'tabpanel', { name: 'Blocks' } )
+				.getByRole( 'option', { name, exact: true } )
```

Comment:
Without exact matching, we couldn't ever select "Form", because it always pulled in "Preformatted" 🤦

Reply:
I filed an issue for the second one. https://github.com/Automattic/wp-calypso/issues/81135

Reply:
Thank you Edwin for testing that! 😄  I ended up not having time on Friday to test all the prior block flows with the change. 

> I filed an issue for the second one. https://github.com/Automattic/wp-calypso/issues/81135 and asked Jeremy for priority.

Looks like he already has a PR for this! 🎉  I'll wait to merge until this is merged.

> The first issue is easy to fix - I'll leave it up to you whether you want to make the one-line change in this PR or I can do it.

I saw you did this in your Ads PR (thank you), and I can either copy it here and/or just rebase before I merge. I'll make sure everything is green on this branch either way before deploying. 😄

Reply:
My Ads PR requires this PR to be merged (I built on top of your changes) so if preferable, let's deploy yours first, and then immediately deploy my Ads PR after.